app/locale/zh-HANS.coffee

```diff
@@ -1077,40 +1077,40 @@ module.exports = nativeDescription: "简体中文", englishDescription: "Chinese
     mission_description_2: "同时，他们也能迅速学习，因为学习的方式变快了。学习就像是对话而不是阅读枯燥的手册。我们想将这种对话带给每所学校和<strong>每个学生</strong>，因为每个人都应有机会学习编程的魔法。"
     team_title: "与CodeCombat团队见面"
     team_values: "我们期待开放的、互相尊敬的对话，好点子会得到采纳。我们决定深入市场调研，我们集中于提供实际性的结果。从我们的 CEO 到 Github 上的贡献者，每个人都亲力亲为，因为在团队中我们收获了成长与经验。"
-#    nick_title: "Cofounder, CEO"
```

Comment:
temporarily no need for translating titles

Reply:
Ok, I will change the code soon.

Reply:
thanks